weave-js/src/components/Sidebar/Outline.tsx

```diff
@@ -112,6 +121,13 @@ const OutlinePanel: React.FC<OutlinePanelProps> = props => {
     level = 0,
     setInspectingRoot,
   } = props;
+
+  const {
+    state: {panelState},
+  } = usePanelInteractContext();
+  const setPanelIsHoveredInOutline = useSetPanelIsHoveredInOutline();
+  const panelIsHovered = panelState[path.join(`.`)]?.hovered ?? false;
```

Comment:
nit: `panelState[path.join(`.`)]` feels like something that could be abstracted away - in particular, feels odd that the caller has to know to join on dots to lookup the state.

Reply:
+1, there's a pattern in PanelInteractContext for "getter" hooks. You should put this logic in there.